apps/wpcom-block-editor/src/wpcom/features/convert-to-blocks-button.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+
+import { addFilter } from '@wordpress/hooks';
+import { BlockControls } from '@wordpress/block-editor';
+import { Toolbar, ToolbarButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/compose';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { rawHandler, serialize } from '@wordpress/blocks';
+/* eslint-enable import/no-extraneous-dependencies */
+
+/**
+ * Internal dependencies
+ */
+
+export function withConvertToBlocksButton( ClassicEdit ) {
+	return class extends ClassicEdit {
+		render() {
+			const defaultBlock = super.render();
+
+			if ( ! Array.isArray( defaultBlock ) ) {
+				return <ClassicEdit { ...this.props } />;
+			}
+
+			return (
+				<>
+					<BlockControls>
+						<Toolbar>
+							<ToolbarButton
+								title={ __( 'Convert to blocks' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Convert to blocks?' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/convert-to-blocks-button)